### PR TITLE
Fix build + update

### PR DESCRIPTION
**Pull request by BachToTheFuture**

### CODE REVIEW

File: src/cd.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ D_CD::D_CD()
 
 	// we need it to be the right type; this will print an error message
 	// if we didn't find it
-	if (param->Type() == BParameter::B_CONTINUOUS_PARAMETER)
+/*	if (param->Type() == BParameter::B_CONTINUOUS_PARAMETER)
```
Comment (waddlesplash) [inline src/cd.cpp]: This is probably important and shouldn't be commented out.

File: src/main.cpp [hunk 2]
```diff
@@ -19,8 +19,8 @@ int main(int argc,char *argv[])
 			hide = true;
 	}
 
-	App		myApp( argc>1 ?  argv[argc-1] : "");	
+	App		myApp(  argv[argc-1]);
```
Comment (waddlesplash) [inline src/main.cpp]: You can change the function signature to be `const char* argv[]` instead of `char* argv[]` and then revert this change.

**Comment by BachToTheFuture** [comment]
@waddlesplash Thanks for reviewing! Applied the suggestions.
